packages/train2go-bridge/test/chrome-mock.js

```
@@ -15,13 +15,23 @@ const chromeMock = {
     onMessageExternal: {
       addListener: vi.fn(),
     },
+    onInstalled: {
+      addListener: vi.fn(),
```

ESLint

[error] 'vi' is not defined.

(no-undef)